src/beaconer.rs

```diff
@@ -265,47 +261,6 @@ impl Beaconer {
                 )
             })
             .await;
-
-        // Disable secondary beacons until TTL is implemented
-        if false {
-            self.handle_secondary_beacon(report).await
-        }
-    }
-
-    async fn handle_secondary_beacon(&mut self, report: poc_lora::LoraWitnessReportReqV1) {
```

Comment:
Ah secondary beacons, how we will miss even the code that showed you once existed.

Reply:
lol.. got too annoying to maintain unused code